src/legacy/core_plugins/kibana/public/discover/controllers/discover.js

```diff
@@ -72,6 +72,10 @@ import { buildVislibDimensions } from 'ui/visualize/loader/pipeline_helpers/buil
 import 'ui/capabilities/route_setup';
 import { addHelpMenuToAppChrome } from '../components/help_menu/help_menu_util';
 
+import { data } from 'plugins/data/setup';
```

Comment:
Sorry for the inconsistency - things are changing rapidly :)

In https://github.com/elastic/kibana/pull/42238, a few weeks ago, we standardized the way `setup` is imported

Please use 
import { setup as data } from '../../../../core_plugins/data/public/legacy';

Reply:
I had to import from [../../../../../core_plugins/data/public/legacy](https://github.com/elastic/kibana/pull/39140/commits/08a36839093b9d59ba238c824a250428d2fb54e9#diff-db6f4af0165205b24c9cdd101a152ddeR75)